### ISSUE TITLE
CAS0 and CAS1 swapped

### ISSUE THREAD

**Issue by Kyp069** [pelrun/cpc-schematics#2]
Hi, first thank you for the project.

I think I have found a mistake:

![imagen](https://github.com/pelrun/cpc-schematics/assets/21128002/a11482b7-1dde-425b-a589-f2edd75b5698)

In IC118:
CAS0 is pin 17
CAS1 is pin 16
But they are swapped in your schematic.
I have also checked against original PCB.

**Comment by pelrun** [comment]
Yeah, they're probably swapped. However the connectivity is right - pin 16 goes to the left hand bank (IC119 and up) and pin 17 goes to the right (IC127 and up).
